src/plugins/intel_cpu/src/nodes/executors/convolution_implementations.cpp

```diff
@@ -49,7 +49,10 @@ static const TypeMapping dnnlConvTypeMapping {
     {{_bf16, _f16, _any, _any},                        pt(bypass(), bypass(), use<0>(), use<0>())},
     {{_f16, _bf16, _any, _any},                        pt(bypass(), bypass(), use<0>(), use<0>())},
     // quantization configuration
-    {{_u8 | _i8, _i8, _quant | _hw_float | _i32 | _dynamic, _quant | _hw_float | _i32 | _dynamic}, pt(bypass(), bypass(), bypass(),  bypass())},
+    // int8 conv does not support f16 output and bias
```

Comment:
Is it the case for oneDNN 3.8 version as well?

Reply:
Hi, @EgorDuplensky : I think so, since the latest upstream oneDNN still have this limitation:
https://github.com/uxlfoundation/oneDNN/blob/5f44f1ec4943d5ac322471e4c0695ec7ab2be89d/src/cpu/x64/jit_brgemm_conv.cpp#L371-L374